tests/reinit.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mysql --host lsst10 -u metaServ -p -e "drop   database metaServ_core"
```

Comment:
server host name is hardcoded, can you use a variable name (and optionally us command line parameter to change it) to make it easier to change?